.github/workflows/ci.yml

```diff
@@ -27,11 +25,12 @@ jobs:
           server-password: OSSRH_TOKEN # env variable for token in deploy
           gpg-private-key: ${{ secrets.MAVEN_CENTRAL_GPG_KEY }} # Value of the GPG private key to import
           gpg-passphrase: MAVEN_GPG_PASSPHRASE # env variable for GPG private key passphrase
-      - name: Publish alfresco-value-assistance-repo packages
```

Comment:
I missed this last time but the name on L19 is incorrect: `Set up Maven Central Repository` I don't think you are setting op a repository :)
 `Set up JDK and Maven options` for example is more correct.